test/e2e/data/capi-operator/capv-variables.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: capv-system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vsphere-variables
+  namespace: vsphere-system
```

Comment:
```suggestion
  namespace: capv-system
```
One nit

Reply:
Nice catch @Danil-Grigorev, fixed it now